src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -567,6 +568,31 @@ public void setPartitionSizeThreshold(@Nullable DataStorageSpec.LongBytesBound w
                                   x -> config.partition_size_fail_threshold = x);
     }
 
+    @Override
+    public long getPartitionTombstonesWarnThreshold()
+    {
+        return config.partition_tombstones_warn_threshold;
+    }
+
+    @Override
+    public long getPartitionTombstonesFailThreshold()
+    {
+        return config.partition_tombstones_fail_threshold;
+    }
+
+    public void setPartitionTombstonesThreshold(long warn, long fail)
+    {
+        validateMaxLongThreshold(warn, fail, "partition_tombstones", false);
+        updatePropertyWithLogging("partition_tombstones_warn_threshold",
```

Comment:
There is one thing I want to discuss, why do we use the name of this variable attribute here， see "partition_tombstones_warn_threshold" ? Because I feel that it is easy for developers to spell mistakes, and if the test does not cover it, there may be some problems。

Or we can use just some map which key is property name value is the property value or some enum class ？

Reply:
Indeed it's not ideal that we don't know the name of the yaml property that will be written to the log output. The original prototype for guardrails had a much more robust system of proper config objects for every guardrail, instead of the current flat system. However, new nested configs were vetoed by a committer until we moved the entire config file to a nested structure, even if we already have some of them in the yaml. We haven't been still able to settle on that new nested format, but I guess that when/if that happens this will be less cumbersome.

I guess that for not repeating the name of the property we could try to read its name from the `Config` var, or something like that. In any case, that looks like a general refactor of the framework to be done in a separate ticket.

Reply:
got it